installing/installing-fips.adoc

```diff
@@ -14,19 +14,67 @@ For more information about the NIST validation program, see link:https://csrc.ni
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} 9 computer that is configured to operate in FIPS mode, and you must use a FIPS-capable version of the installation program. See the section titled _Obtaining a FIPS-capable installation program using `oc adm extract`_.
 
-For more information about configuring FIPS mode on {op-system-base}, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening[Installing the system in FIPS mode].
+For more information about configuring FIPS mode on {op-system-base}, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
 ====
 
 For the {op-system-first} machines in your cluster, this change is applied when the machines are deployed based on the status of an option in the `install-config.yaml` file, which governs the cluster options that a user can change during cluster deployment. With {op-system-base-full} machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines.
 
 Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
 
+[id="installation-obtaining-fips-installer-oc_{context}"]
+== Obtaining a FIPS-capable installation program using `oc adm extract`
+
+{product-title} requires the use of a FIPS-capable installation binary to install a cluster in FIPS mode. You can obtain this binary by extracting it from the release image using `oc`. After you have obtained the binary, you proceed with the cluster installation, replacing all instances of the `openshift-install` command with `openshift-install-fips`.
+
+.Prerequisites
+
+* You have installed the `oc` command line utility with version 4.16 or newer.
+
+.Procedure
+
+. Extract the FIPS-capable binary from the installation program by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract --registry-config "${pullsecret_file}" --command=openshift-install-fips --to "${extract_dir}" ${RELEASE_IMAGE}
+----
++
+where:
++
+--
+`<pullsecret_file>`:: Specifies the name of a file that contains your pull secret.
+`<extract_dir>`:: Specifies the directory where you want to extract the binary.
+`<RELEASE_IMAGE>`:: Specifies the Quay.io URL of the {product-title} release you are using. For more information on finding the release image, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#retrieving-the-openshift-installer_ipi-install-installation-workflow[Extracting the OpenShift Container Platform installation program].
+--
```

Comment:
No link in a module. Unless you know something that I don't (quite likely!).

Reply:
Thank you. I noticed that the link was indeed in an assembly. Sorry for doubting you. I think because it was in a procedure, I knee-jerk assumed it was a module!